### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by uneverknwwhoim** [MITESHPUTHRANNEU/Speech-Emotion-Analyzer#2]
![image](https://user-images.githubusercontent.com/32920949/45252023-f94a3f80-b36d-11e8-9d80-e9f6bd3e97ba.png)
Check the image. This is the error i am getting when i run! plz reply asap

**Comment by MITESHPUTHRANNEU** [comment]
mylist[1800] is one of the audio files from the dataset. Check if you have downloaded all the data and have 1800 audio files. Or else you can use 180. 
Also are you trying to run the python notebook as .py file?

**Comment by Wintermute4** [comment]
Hi! Thanks for posting this, it is a very interesting demo case!
The only issue is that label encoding is performed during the training of the model - so there is no way to see what the labels are unless you run the full thing using all the data (for example I get "5" as an output for male_angry). I only want to load the model just to see if it works as a demo case but I can't decode the outputs from int to labels. Downloading everything just for that purpose seems quite tedious. Would it be possible to get the labels somehow? Thank you!

**Comment by MITESHPUTHRANNEU** [comment]
@Wintermute4, Thank you for bringing this up. Other people have requested for the clarification as well, so I will update the README file.
The list is as follows:

0 - female_angry
1 - female_calm
2 - female_fearful
3 - female_happy
4 - female_sad
5 - male_angry
6 - male_calm
7 - male_fearful
8 - male_happy
9 - male_sad

Hope this helps.

**Comment by Wintermute4** [comment]
Thank you, much appreciated!

**Comment by uneverknwwhoim** [comment]
> 
> 
> mylist[1800] is one of the audio files from the dataset. Check if you have downloaded all the data and have 1800 audio files. Or else you can use 180.
> Also are you trying to run the python notebook as .py file?

Yes i am running as .py file

**Comment by uneverknwwhoim** [comment]
> 
> 
> mylist[1800] is one of the audio files from the dataset. Check if you have downloaded all the data and have 1800 audio files. Or else you can use 180.
> Also are you trying to run the python notebook as .py file?

i have downloaded the zip file , extracted it and then i have open the jupyter nookbook and saved the file as .py and have run the program

**Comment by uneverknwwhoim** [comment]
> 
> 
> Thank you, much appreciated!

Can u help me in running the program?

**Comment by MITESHPUTHRANNEU** [comment]
@uneverknwwhoim you can't rename the notebook and run it as a python file. There are things that happen differently than traditional line of codes. Kindly download [Anaconda](https://www.anaconda.com/download/). It includes the jupyter notebook where you can open the notebook available in the repository. 

You need to have a system with Nvidia GPU and make sure you install all the required libraries like [Keras](https://keras.io/), [CUDA ](https://www.tensorflow.org/install/install_windows)libraries with Tensorflow backend and [Librosa](https://librosa.github.io/librosa/)

Also there are two different things you can do here. 1. Train a new model or 2. Use the trained model. 
If you want to use the trained model then you can start with cell number 137 and then continue from 485. The input audio file at cell number 485 is the audio file that has been recorded from AudioRecorder notebook.

**Comment by uneverknwwhoim** [comment]
I have installed anaconda and i have opened jupyter notebook after that how
will i get the output screen showing results ? And one more thing how can i
use my data (if i have audio files like around 8k wav files) and check the
output.

**Comment by MITESHPUTHRANNEU** [comment]
Just run each each and every cell to see the outputs what they show.

**Comment by uneverknwwhoim** [comment]
So thats the output? And how can i input if i have my own dataset of 8k
Audio files?

**Comment by MITESHPUTHRANNEU** [comment]
See the program. It clearly states where it is importing the datasets. Just change the path to your location.

**Comment by uneverknwwhoim** [comment]
So we need to run this program in jupyter notebook only? Not normally in
python file?

**Comment by MITESHPUTHRANNEU** [comment]
Yes

**Comment by islamoc** [comment]
@MITESHPUTHRANNEU  what if I wanted to use just the model without the training part there is the LabelEncoder that is constructed in the training phase how can I deal with that ?

**Comment by MITESHPUTHRANNEU** [comment]
@islamoc, If you want to directly use the model, I have included the encoders in the project's readme file. 

Also I have answered it above.

**Comment by islamoc** [comment]
I figured it out yes thank you

**Comment by uneverknwwhoim** [comment]
where can i find the dataset of audio files? and i am not able to find the dataset of audiofiles from your project?

**Comment by Nandy-Saran** [comment]
The dataset is RAVDESS and SAVEE. The links are in the READme file itself. You just need to register to download the SAVEE database. The RAVDESS is free.
